guillotina/behaviors/properties.py

```diff
@@ -10,19 +10,11 @@ def __get__(self, inst, klass):
         if inst is None:
             return self
 
-        if hasattr(inst.context, self.__name__):
```

Comment:
Why we are removing this check ?

Reply:
Because we always want to set the value even if it didn't previously have a value on the object.

Reply:
Isn't it that in case that we have the AttributeError we were displaying the default value ? Setting it will make things harder to change the default value of a field.

Reply:
We are using the default value here though `self.default`